### DISCUSSION TITLE
Screen problem

### DISCUSSION THREAD

**Discussion by Pabcandy** [eh2k/squares-and-circles#86] [General]
Hi,
Fantastic work. I have the Jakplugg uOC with the DAC8565 with Teensy 4 which worked with the Phazerville. After flashing Squares and Circles it initially worked fine but messing around I accidentally clicked on the DAC8565/DAC8564 button on the calibration screen and I think it may have caused a terrible glitch with the screen when a trigger is input into a trigger jack. The screen goes very fuzzy and is unviewable. Do you know if I am likely to have fried something and if so, what? I might try replacing the DAC but they are like £15 !
thanks

**Comment by eh2k** [comment]
Hi, changing the settings will only change the way the dac is controlled. In the worst case, there is no output at the cv outputs. Click again to switch between the modes. 

Please try to switch between the display modes "DISPLAY_SPI/SPI1".  Maybe you have activated the alternative display mode?. 

Apart from that I don't know how you can damage the hardware - except if the Teensy is connected via USB and at the same time supplied with power from the Eurorack, if the VIN trace on the teensy is not cut.

**Reply by Pabcandy** [reply]
Hey,
I managed to get it sorted. You'll laugh but the aluminium panel I had built was shorting the screen connectors at the top. Took me ages to figure it out but a bit of tape sorted it.
Thanks once again for your fantastic work - it really is appreciated. Without people like you in the world, we'd still be living in caves.
Phil
